HoloPipelines/pipelines/components/compHttpRequest.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import pathlib
+import sys
+import logging
+
+
+def sendFilePostRequest(url, inputFile, outputFile):
+    inputFile = str(pathlib.Path(inputFile))
+    outputFile = str(pathlib.Path(outputFile))
+    with open(inputFile, "rb") as inputFileData:
+        fileToSend = {"file": inputFileData}
+        try:
+            response = requests.post(url, files=fileToSend, timeout=10)
+            if response.status_code != 200:
+                sys.exit(
+                    "compHttpRequest: error, bad status code (got {})".format(
```

Comment:
Is there any other information that will be useful when such a situation occurs? Does the server respond with anything useful in the response body that could be printed as well?

Reply:
Updated error msg with content of the response from the server (the other status code aside from 200 is 400 with "No file in the request" or "No selected file" as the content)